doc/lua_api.txt

```diff
@@ -3402,8 +3402,45 @@ You can find mod channels communication scheme in `docs/mod_channels.png`.
     * Returns `nil` if the given `paramtype2` does not contain color
       information.
 * `minetest.get_node_drops(nodename, toolname)`
-    * Returns list of item names.
+    * Returns list of itemstrings.
     * **Note**: This will be removed or modified in a future version.
+* `minetest.get_extended_drops(drop_table, toolname, palette_index)
+    * Calculates the item(s) to drop, given the drop_table and the tool.
+    * palette_index is optional, see Hardware coloring for more detail.
+    * drop_table looks like:
+        drop_table = {
```

Comment:
Should be formatted as multiline code. I.e.
\`\`\`
multiline
code
\`\`\`
This should also be moved to the bottom of this file and referenced as "Drop definition table" (or similar) here and in "Node definition". Otherwise we're documenting the same thing two times.